fitlins/interfaces/bids.py

```diff
@@ -523,7 +430,9 @@ def _list_outputs(self):
             ents.update(entities)
             ext = bids_split_filename(in_file)[2]
             ents['extension'] = self._extension_map.get(ext, ext)
-
+            ents.update({k: str(v).replace('.', '_')
```

Comment:
A comment here would help.

Reply:
I have sprinkled in more comments where I thought they were needed. Lmk if something is still unclear.